.changeset/fancy-dancers-draw.md

```diff
@@ -0,0 +1,15 @@
+---
+"@fluidframework/telemetry-utils": minor
+"@fluidframework/odsp-driver": minor
+"@fluidframework/runtime-definitions": minor
+---
+
+Upcoming: The type of the logger property/param in various APIs will be changing
```

Comment:
Note that these will end up in the CHANGELOG.md for the listed packages. I think that's ok, just FYI.